content/devdocs/references/p2p_networking.md

```diff
@@ -218,7 +309,23 @@ The following annotated hexdump shows an `ANNT` message with two
 inventory entries.  (The message header has been omitted.)
 
 {{ <highlight text> }}
-ANNT message hex dump
+AnnounceTxnsMessage:
+```
+48 00 00 00 00 00 00 00 02 00 00 00 83 d5 44 cc 
+c2 23 c0 57 d2 bf 80 d3 f2 a3 29 82 c3 2c 3c 0d 
+b8 e2 67 48 20 da 50 64 78 3f b0 97 83 d5 44 cc 
+c2 23 c0 57 d2 bf 80 d3 f2 a3 29 82 c3 2c 3c 0d 
+b8 e2 67 48 20 da 50 64 78 3f b0 97 ............... Full message
+------------------------------------------------------------------------
+0x0000 48 00 00 00 00 00 00 00 ........................... Prefix
```

Comment:
This is not quite like it should be . First four bytes are the total size of the buffer that follows . The other four bytes are the message `Prefix` ID (e.g.one of `INTR`, `GETB`, ...) . Please fix that